lib/screens/lobby.dart

```diff
@@ -32,26 +37,32 @@ class LobbyScreen extends StatelessWidget {
           )
         ],
       ),
-      body: Container(
-        child: StreamBuilder<List<Game>>(
-            stream: context.watch<DatabaseService>().streamOfCreatedGames,
-            builder: (context, snapshot) {
-              if (snapshot.data == null) {
-                return Container();
-              } else {
-                return Padding(
-                  padding: const EdgeInsets.fromLTRB(0, 10, 0, 0),
-                  child: ListView.builder(
-                    itemCount: snapshot.data.length,
-                    itemBuilder: (context, index) {
-                      return GameTile(
-                        snapshot.data[index],
-                      );
-                    },
-                  ),
-                );
-              }
-            }),
+      body: FutureBuilder<Position>(
+        future: context.read<LocationService>().getUsersCurrentLocation(),
+        builder: (context, snapshot) => snapshot.hasData == false
```

Comment:
this future builder is really all this pr is,
in the origianl pr (#60) ive cleaned up this file so please avert your eyes for now